apps/sms/views/conversation/js/attachment.js

```diff
@@ -26,7 +26,8 @@
   function Attachment(blob, options) {
     options = options || {};
     this.blob = blob;
-    this.name = blob.name || options.name || '';
+    this.name = typeof blob.name == 'string' ? blob.name : (
+      typeof options.name == 'string' ? options.name : '');
```

Comment:
@julienw  ah ah. your code is so clean like alwz :smiley: 
if (typeof this.name !== 'string') { this.name = '' } -> okay with it